pkg/util/k8s/node.go

```diff
@@ -57,3 +59,63 @@ func GetNodeAddrs(node *v1.Node) (*ip.DualStackIPs, error) {
 	}
 	return nodeAddrs, nil
 }
+
+// GetNodeAddressFromAnnotations gets available IPs from the Node Annotation.The annotations are set by Antrea, including
+// NodeTransportAddressAnnotationKey string = "node.antrea.io/transport-addresses"
```

Comment:
As a generic function, its comment shouldn't be specific an annotation key, or you should just remove the second parameter.
And please name the functions in same style:
GetNodeAddrs
GetNodeAddrsFromAnnotations
GetNodeGatewayAddrs

Reply:
Updated.